modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/ClassicEditor.js

```diff
@@ -13,27 +13,16 @@
  */
 
 import {useEventListener} from 'frontend-js-react-web';
-import {debounce, isPhone, isTablet} from 'frontend-js-web';
+import {debounce} from 'frontend-js-web';
 import PropTypes from 'prop-types';
 import React, {useCallback, useEffect, useRef, useState} from 'react';
 
 import {Editor} from './Editor';
 
-const getToolbarSet = (toolbarSet) => {
-	if (isPhone()) {
-		toolbarSet = 'phone';
-	}
-	else if (isTablet()) {
-		toolbarSet = 'tablet';
-	}
-
-	return toolbarSet;
-};
-
 const ClassicEditor = ({
 	contents = '',
 	editorConfig,
-	initialToolbarSet,
+	initialToolbarSet = 'simple',
```

Comment:
If I understand correctly, the intent is to make `simple` toolbar be the default. But, if we use the component through `liferay-editor:editor` tag, we set `initialToolbarSet` to `liferay`, [here](https://github.com/liferay/liferay-portal/blob/3294c1b9117f72b6f13a53ca1e1ff944be28397c/util-taglib/src/com/liferay/taglib/ui/InputEditorTag.java#L465). This will not have apparent effect as they are both the same, but for clarity we could:
- Change default in `InputEditorTag`. I am not sure if that is safe to do, what the scope of the class is.
- Remove the `initialToolbarSet` prop and only rely on toolbar sets set through `config`, with `simple` as default:
```
	const getConfig = () => {
		return {
			toolbar: 'simple',
			...editorConfig,
		};
	};
```